homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -90,7 +90,7 @@ def _connect(self) -> None:
 
     def handle_event(self, response):
         """Handle responses from the speakers."""
-        data = response["data"] if "data" in response else {}
+        data = response.get("data", {})
```

Comment:
This one is debatable actually...

The new version, will always create an empty dict to pass into that methods, the original one, will only create the dict if it actually gets there...

The new version is less efficient.

Reply:
True, although I don't think it really matters in this case.

Technically the old code does two dict lookups whereas `.get` only does one 🤷🏻‍♂️
I can revert that if you like.

Reply:
One example I did exclude was this
https://github.com/home-assistant/core/blob/c82933175dd76b95e30fd847ab0f03d45fce883b/homeassistant/components/sensor/recorder.py#L150-L152

Here `unit` can be `None`, so the contains check is at least type safe even though get should provide the same result. _`None` just isn't a key in the dict, so it should fallback to the default in that case._

Reply:
I wonder, is this more efficient?

```
data = response.get("data") or {}
```

Reply:
> I wonder, is this more efficient?
> 
> ```
> data = response.get("data") or {}
> ```

Maybe ever so slightly. I like it nevertheless. Pushed a new commit.

Reply:
The only difference is that `response.get("data", {})` will return `None` if it is explicitly set to `None` in the dict, and `data = response.get("data") or {}` will always return `{}`.

Reply:
> The only difference is that `response.get("data", {})` will return `None` if it is explicitly set to `None` in the dict, and `data = response.get("data") or {}` will always return `{}`.

👍🏻 From what I can tell returning an empty dict is fine in this case. But yes, that's something to look out for in future instances.
https://github.com/home-assistant/core/blob/bf8bd5ff21c665d2c76b6fdf5153f4157c489942/homeassistant/components/lg_soundbar/media_player.py#L91-L98